config.py

```diff
@@ -21,20 +27,19 @@ def __init__(self):
         self.mac_addrs = config_json["mac_addrs"]
         self.alert_collection_path = config_json["alert_collection_path"]
         self.virustotal_api_key = config_json["virustotal_api_key"]
-        with open('/etc/tinyHIPPO/cids-startuo.log', 'a+') as logging_file:
-            smtp_server = config_json['email']['smtp_server']
-            email_account = config_json['email']['email_account']
-            email_key = config_json['email']['email_password']
-            recipient_email = config_json['email']['recipient_email']
-
-            if smtp_server == "smtp.example.com":
-                logging_file.write('STMP Server not configured\n')
-
-            if email_account == "openwrt@example.com":
-                logging_file.write('EMAIL ACCOUNT NOT CONFIGURED\n')
-
-            if email_key == "super_secure_password":
-                logging_file.write('Email account password not set\n')
-
-            if recipient_email == "homeowner@example.com":
-                logging_file.write('Recipient email not configured\n')
+        self.log_event = logging.getLogger()
+        
+        smtp_server = config_json['email']['smtp_server']
+        email_account = config_json['email']['email_account']
+        email_key = config_json['email']['email_password']
+        recipient_email = config_json['email']['recipient_emails']
+
+        if smtp_server == "smtp.example.com":
+            self.log_event.info('STMP Server not configured\n')
+        if email_account == "openwrt@example.com":
+            self.log_event.info('EMAIL ACCOUNT NOT CONFIGURED\n')
+        if email_key == "super_secure_password":
+            self.log_event.info('Email account password not set\n')
+        if recipient_email[0] == "homeowner@example.com":
+            self.log_event.info('Recipient email not configured\n')
+            
```

Comment:
Please remove this logic, I'll explain in our next meeting but I don't think it's necessary for us to check template values for the user in this portion of the code.